### DISCUSSION TITLE
Can't add lua-languafe-server

### DISCUSSION THREAD

**Discussion by Esnos33** [helix-editor/helix#6839] [General]
Hi, I want to write code in lua, but I have problem with adding lsp to helix.

Code bellow shows my problem:

hx --health lua
Configured language server: lua-language-server
Binary for language server: 'lua-language-server' not found in $PATH
Configured debug adapter: None
Highlight queries: ✓
Textobject queries: ✓
Indent queries: 

And I was reading this [installation guide](https://github.com/LuaLS/lua-language-server/wiki/Getting-Started) and after "./bin/lua-language-server " I get this in terminal

~/lua-language-server$ ./bin/lua-language-server 
Content-Length: 120

{"jsonrpc":"2.0","method":"$/status/report","params":{"text":"😺Lua","tooltip":"Cached files: 0/0\nMemory usage: 2M"}} 
lisek@pop-os:~/lua-language-server$ cd ..

What I should do to make lua work in helix?

**Comment by kirawi** [comment]
You have to add the binary to your `PATH` for Helix to be able to find it.

**Reply by Esnos33** [reply]
So I have something like this:
echo $PATH
/home/\<user\>/lua-language-server/
But lsp still doesn't work, can you help me out? I'm linux noob.

**Reply by kirawi** [reply]
You have to have it point to the directory containing the binary.

**Reply by Esnos33** [reply]
I have it! I had to add export PATH="/home/\<user\>lua-language-server/bin/:$PATH" for it to works!
Last question, do you know where I can learn about binaries path etc.? And why sometimes I can just use make install, and sometimes not? Also, I have this folder under \<user\>, this is the best place for it, or should I have it somewhere else?
